### PR TITLE
Update: path to ArcGIS xcframework

### PR DESCRIPTION
**Pull request by yo1995**
This PR updates the `v.next` branch to use the latest `xcframework` paths as well as the [`update10`](https://github.com/Esri/arcgis-runtime-toolkit-ios/tree/update10) branch of the toolkit dependency.

Please refer to https://github.com/Esri/arcgis-runtime-toolkit-ios/pull/122

Please notice: after being merged into `v.next`, one can no longer build with an SDK prior to 2957, since the paths are changed.

### CODE REVIEW

File: arcgis-ios-sdk-samples.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1626,7 +1626,7 @@
 		4C72116F20BF444C004A7DD9 /* SVProgressAnimatedView.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = SVProgressAnimatedView.h; sourceTree = "<group>"; };
 		4C72117120BF4456004A7DD9 /* SVRadialGradientLayer.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = SVRadialGradientLayer.h; sourceTree = "<group>"; };
 		4C72117220BF4457004A7DD9 /* SVRadialGradientLayer.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = SVRadialGradientLayer.m; sourceTree = "<group>"; };
-		4C73C83421112B4300B0F9C7 /* ArcGIS.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = ArcGIS.framework; path = $HOME/Library/SDKs/ArcGIS/iOS/Frameworks/Dynamic/ArcGIS.framework; sourceTree = "<absolute>"; };
+		4C73C83421112B4300B0F9C7 /* ArcGIS.xcframework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.xcframework; name = ArcGIS.xcframework; path = $HOME/Library/SDKs/ArcGIS/Frameworks/ArcGIS.xcframework; sourceTree = "<absolute>"; };
```
Comment (yo1995) [inline arcgis-ios-sdk-samples.xcodeproj/project.pbxproj:1629]: image

I think the path should be absolute, but in toolkit it is set to "Relative to Group".

**Review by yo1995 — COMMENTED** [review]
One uncertainty below.

**Review by vquach2404 — APPROVED** [review]

File: arcgis-ios-sdk-samples.xcodeproj/project.pbxproj (hunk 1 above)
Comment (philium) [reply arcgis-ios-sdk-samples.xcodeproj/project.pbxproj:1629]: You are correct; it should be absolute. That should be corrected in the toolkit.

**Comment by saratchandrakarumuri** [comment]
I am getting build errors with toolkit due to that relative path

2020-11-12 at 11 11 AM

File: arcgis-ios-sdk-samples.xcodeproj/project.pbxproj (hunk 1 above)
Comment (yo1995) [reply arcgis-ios-sdk-samples.xcodeproj/project.pbxproj:1629]: OK. Let me create a PR to fix that.

Comment (yo1995) [reply arcgis-ios-sdk-samples.xcodeproj/project.pbxproj:1629]: I don't have write access to toolkit repo. Let me create an issue instead.

Comment (philium) [reply arcgis-ios-sdk-samples.xcodeproj/project.pbxproj:1629]: I can fix it.

Comment (philium) [reply arcgis-ios-sdk-samples.xcodeproj/project.pbxproj:1629]: The issue has been [fixed](https://github.com/Esri/arcgis-runtime-toolkit-ios/pull/126).

**Review by saratchandrakarumuri — APPROVED** [review]
Builds successfully with latest build

**Review by vquach2404 — APPROVED** [review]

**Comment by yo1995** [comment]
@saratchandrakarumuri I'm not sure f7dfc51696f9cc8471568b34c724e2d5eb3fe412 is the reason of your build error, but please give another try. Thanks!

**Review by saratchandrakarumuri — APPROVED** [review]
This fixed the issue... I am able to launch app and run bunch of samples without any issue

**Review by philium — APPROVED** [review]

**Pull request merged**